### ISSUE TITLE
Copy in "Import Community" dialogue in Settings is incorrect

### ISSUE THREAD

**Issue by John-44** [status-im/status-desktop#8339]
see this screenshot below that shows 0.8 RC2 next to the Figma design for this screen

![2022-11-18 23_23_44-⚙️ Settings _ Desktop – Figma](https://user-images.githubusercontent.com/22846356/202795972-2d38689f-4e93-406f-a30f-d3cdf15b55cd.png)

The copy should say "Enter the public key of the community you wish to access, or enter the private key of a community you own. Remember to always keep any private key safe and never share a private key with anyone else."

And users should be able to enter both public and private community keys into this same dialogue as per https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=1369%3A136806&t=5NOZLQogKXxpxAGz-1

**Comment by PascalPrecht** [comment]
Have we already discussed somewhere how we're retrieving the community information just by the public key, assuming that the community wasn't advertised for the user anywhere?

**Comment by John-44** [comment]
@PascalPrecht Thanks for your message.  I agree that it would be better if: when a user enters the public key of a community and they click the "Import" button, they are taken to the home screen of the community where they can then press the 'Join' or 'Request to join' button.  This simplifies things and is also a more consistent experience for the user

**Comment by PascalPrecht** [comment]
Thanks @John-44 
Here's another thing that needs clarification:

Judging by the designs, when a public or private key is entered and is valid, there's some community information shown below the textarea, even though the "import" buttons hasn't been clicked yet.
![Screenshot from 2022-11-28 12-39-14](https://user-images.githubusercontent.com/445106/204269106-6ae81ee6-7564-46e4-b3b4-4678809ef1ba.png)

We don't have that information magically available and need to request it first. This is going to be an async operation where showing a loading indicator makes sense.

Should automatically fetch community info when the public/private key is entered, or should this only happen after the "import community" button has been clicked?

If we postpone fetching that information to after the "Import community" button has been clicked, we can wait for the response and then turn the button into the "Make this an Owner node" and "Go to Community" respectively.

In other words: first load community info, **then** decide whether something should be done.

**Comment by John-44** [comment]
@PascalPrecht This interface works without fetching this information after a key is entered, so for expediency's sake (and because we have a lot of other work that needs to be done) let's remove this from the MVP.

**Comment by anastasiyaig** [comment]
Acceptance criteria:

1. user should be able to import community with community public key (setting screen and community portal still)
2. user should be able to import community with community private key (setting screen and community portal still)
3. fetching information about the community is delayed (post MVP)
4. validation is present when entering invalid key
5. modal matches the designs https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=1369%3A136806&t=NxxOZgd3w53blLDE-4

**Comment by PascalPrecht** [comment]
@anastasiyaig fetching information about the communit (so point 3) is actually implemented already. May I ask what specifically you're referring to?

**Comment by anastasiyaig** [comment]
i was just trying to summarise what actually is being implemented. We are trying to provide this information for issues that are delivering features (so called acceptance criteria). So issue should have this (ideally before development :D) therefore i tried to outline it. Thanks for clarifying  that for me , seems i was lost a bit in the comments thread so i misunderstood :)
